js/main.js

```diff
@@ -149,18 +127,19 @@ document.addEventListener('DOMContentLoaded', function (event) {
     swapView('profile');
   }
 });
-var $modal = document.querySelector('#modal');
 
 document.addEventListener('click', function (event) {
-  if (event.target.tagName === 'A' && event.target.className !== 'header-profile') {
-    swapView('edit-profile');
-  } else if (event.target.className === 'header-profile' && data.profile.username.length !== 0) {
+  if (event.target.tagName !== 'A') {
+    return false;
+  } else if (event.target.getAttribute('data-view') === 'profile') {
```

Comment:
Instead of this being an `else if` statement, you should make it its own separate conditional below the above conditional 👍